cmp/grid/Grid.js

```diff
@@ -561,6 +561,23 @@ class LocalModel extends HoistModel {
         };
     }
 
+    validationDisplayReaction() {
+        const {model} = this,
+            {store} = model;
+
+        return {
+            track: () => [model.isReady, store.isValidationPending],
```

Comment:
this relies on the validation briefly flickering the pending state -- just noting this -- seems brittle